components/bank/Nonprofits.js

```diff
@@ -68,7 +68,7 @@ export default function Nonprofits() {
         <Heading variant="title">Nonprofit? No problem.</Heading>
         <Text variant="lead" color="muted">
           Hack Club Bank is a powerful, safe, and easy-to-use financial
-          platform, whether you're receiving your first donation or spend
+          platform, whether you&apos;re receiving your first donation or spend
```

Comment:
Curious, how come we need to do this? Personally I think it just makes the code less readable.

Reply:
It basically prevents problems surrounding [misplaced angle brackets](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/no-unescaped-entities.md), but yeah, I'll remove it